ruma-api/tests/ui/03-move-value.rs

```diff
@@ -1,6 +1,9 @@
 use ruma_api::ruma_api;
 use ruma_identifiers::UserId;
 
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub struct Foo;
+
 ruma_api! {
```

Comment:
Why is this one not in its own module. Also, it seems more logical to me to have one module with `body` request + `body` response, and one with `raw_body` request + `raw_body` response.